### PR TITLE
Ft code improvement

### PR DESCRIPTION
**Pull request by mwafrika**
- Improved the logic based on the feedback
- Add account verification
- Add reset password template
- Modified relationship in shifts(from one-to-to to many-to-many) relationship

### CODE REVIEW

File: src/controllers/auth.js [hunk 1]
```diff
@@ -166,26 +206,53 @@ export const resetPassword = async (req, res) => {
 };
 
 export const getUsers = async (req, res) => {
-  const { companyId } = req.user;
+  const { companyId, role } = req.user;
+  const roleByName = await findRoleByName(role);
+
+  console.log(req.user, "req.user xxxxxxxx");
```
Comment (Irirwanirira) [inline src/controllers/auth.js]: remove console.log from your codebase

File: src/controllers/shift.js [hunk 2]
```diff
@@ -4,28 +4,31 @@ import {
...
+      employee, startDate, endDate, startTime, endTime
     } = req.body;
-    const user = await findUserById(userId);
-    if (!user) {
-      return formatResponse(res, StatusCodes.NOT_FOUND, null, "User not found");
-    }
+    // const user = await findUserById(userId);
+    // if (!user) {
+    //   return formatResponse(res, StatusCodes.NOT_FOUND, null, "User not found");
+    // }
```
Comment (Irirwanirira) [inline src/controllers/shift.js]: remove commented codes

File: src/database/models/user.js [hunk 3]
```diff
@@ -7,15 +7,36 @@ module.exports = (sequelize, DataTypes) => {
...
         as: "company"
       });
-      User.hasMany(models.Shift, {
-        foreignKey: "userId",
-        as: "shifts"
-      });
+      // User.hasMany(models.Shift, {
+      //   foreignKey: "userId",
+      //   as: "shifts"
+      // });
```
Comment (Irirwanirira) [inline src/database/models/user.js]: same here

**Review by Irirwanirira — CHANGES_REQUESTED** [review]

**Review by Irirwanirira — APPROVED** [review]
Good to go!

**Review by jefftrojan — APPROVED** [review]

**Pull request merged**